client/src/main/java/com/battleships/gamewindow/GameWindowController.java

```diff
@@ -90,27 +90,34 @@ private void bindGUIComponents() {
     private void initBoardService(EventHandler<ActionEvent> shotEvent) {
         BoardSize boardSize = new BoardSize(10, 10);
         boardService = new BoardService(boardSize);
+        Connection.INSTANCE.boardService = boardService;
 
         BoardGridPanes boardGridPanes = new BoardGridPanes(playerGridPaneBoard, opponentGridPaneBoard);
         boardService.initBoards(boardGridPanes, shotEvent);
     }
 
     private void shot(ActionEvent event) {
         BoardField clickedButton = (BoardField) event.getSource();
-        Coordinate coordinate = clickedButton.getCoordinate();
-        clickedButton.refreshColor(); // TODO 24/07/18 damian -  is it neeed
+        Coordinate[] coordinates = new Coordinate[]{clickedButton.getCoordinate()};
+//        clickedButton.refreshColor(); // TODO 24/07/18 damian -  is it neeed?
 
-        sendShootMessageToServer(coordinate);
+        Shot shot = new Shot(coordinates);
+        sendShootMessageToServer(shot);
         makePlayerInactiveAndUnreadyAfterShoot();
 
-        // TODO 24/07/18 damian - AFTER SERVER RESPONS... DO SOMETHING
-        boardService.onShootOpponentMessageRecieve(coordinate, new HitMastField(coordinate));
+        // TODO 24/07/18 damian - AFTER SERVER RESPONS... DO SOMETHING - test method
+        boardService.onShootOpponentMessageRecieve(coordinates[0], FieldState.BUFFER);
```

Comment:
is that needed on master?

Reply:
Yes, it simulates fake response from server about the field on opponent board you shot.
In the future instead of FieldState.BUFFER there will be an actual field that was shot in opponents board

Reply:
so repair it :P it doesnt work